functions/zoom-downloader.py

```diff
@@ -250,7 +249,9 @@ def stream_file_to_s3(file, uuid, track_sequence):
         stream, zoom_name = get_connection_using_download_url(file['download_url'])
 
     metadata['file_type'] = zoom_name.split('.')[-1]
-    filename = create_filename(file['id'], file['meeting_id'], zoom_name)
+    filename = create_filename("{}-{}".format(track_sequence, file['id']),
```

Comment:
Do you know how the concat operation sorts the filenames? I suspect it's lexical, which means this won't do what we want if the list of files is > 9 (probably unlikely but still).

```
In [1]: ids = [str(x) for x in range(1,11)]

In [2]: sorted(ids)
Out[2]: ['1', '10', '2', '3', '4', '5', '6', '7', '8', '9']
```

Could you change the integers to letters? Then we'd be safe for <= 26 files.

```
from string import ascii_uppercase
...
filename = create_filename("{}-{}".format(ascii_uppercase[track_sequence - 1], file['id']), ...
```

Reply:
Instead of letters, changed to 3 digit ints. So 009, 010, 011, etc.